### ISSUE TITLE
Compiler error, MissingFormatArgumentException with Kotlin 1.0.0

### ISSUE THREAD

**Issue by PaoloConte** [robovm/robovm#1234]
Hello!
I'm having an issue at compile time with the latest kotlin releases (1.0.0, 1.0.0-rc-1036 and 1050) and RoboVM 1.13
The exception below is caused by the `String.format` function, such as 

```
"Click Nr. %d".format(++clickCount)
```

```
[ERROR] Couldn't compile app
java.util.MissingFormatArgumentException: Format specifier '%d'
    at java.util.Formatter.format(Formatter.java:2519)
    at java.util.Formatter.format(Formatter.java:2455)
    at java.lang.String.format(String.java:2940)
    at org.robovm.idea.RoboVmPlugin.log(RoboVmPlugin.java:487)
    at org.robovm.idea.RoboVmPlugin.logDebug(RoboVmPlugin.java:483)
    at org.robovm.idea.RoboVmPlugin$4.debug(RoboVmPlugin.java:514)
    at com.robovm.debug.compiler.DebugInfoPlugin.a(Unknown.java:719)
    at com.robovm.debug.compiler.DebugInfoPlugin.afterMethod(Unknown.java:499)
    at org.robovm.compiler.ClassCompiler.compile(ClassCompiler.java:802)
    at org.robovm.compiler.ClassCompiler.compile(ClassCompiler.java:297)
    at org.robovm.compiler.AppCompiler.compile(AppCompiler.java:301)
    at org.robovm.compiler.AppCompiler.compile(AppCompiler.java:384)
    at org.robovm.compiler.AppCompiler.compile(AppCompiler.java:444)
    at org.robovm.compiler.AppCompiler.build(AppCompiler.java:773)
    at org.robovm.idea.compilation.RoboVmCompilerThread.doCompile(RoboVmCompilerThread.java:53)
    at org.robovm.idea.compilation.RoboVmCompilerThread.run(RoboVmCompilerThread.java:59)
```

A test project is attached, same code works with previous kotlin versions

[TestApp.zip](https://github.com/robovm/robovm/files/123631/TestApp.zip)

Thanks

**Comment by badlogic** [comment]
Thanks for reporting, we are looking into it.

**Comment by badlogic** [comment]
The fix will be available in the next RoboVM nightly build 1.13.1-SNAPSHOT in a few hours from now. Thanks for reporting. See #1238 for more information.
